src/views/splash/splash.jsx

```diff
@@ -359,10 +153,10 @@ var Splash = injectIntl(React.createClass({
             'teacherbanner.resourcesButton': formatMessage({id: 'teacherbanner.resourcesButton'}),
             'teacherbanner.faqButton': formatMessage({id: 'teacherbanner.faqButton'})
         };
-        if (this.state.projectCount === this.getInitialState().projectCount) {
+        if (this.props.projectCount.projectCount === countActions.getInitialState().projectCount) {
```

Comment:
Here I see why it's being exported... it's weird we were calling `getInitialState()` at all — I think it would be cleaner to let getInitialState be something that only the reducer uses, and if other things need access to values from `getInitialState`, those values should be constants used by `getInitialState` and elsewhere. E.g, add a constant to `countActions` called `DEFAULT_PROJECT_COUNT`, and use that here and in `getInitialState`.